manifests/capture.archons.yaml

```diff
@@ -0,0 +1,16 @@
+﻿name: capture.archons
```

Comment:
by the way: there's no naming convention that requires lowercase separated by dots. Just thought i'd mention it.